packages/logic-utils/package.json

```diff
@@ -21,7 +21,7 @@
     "@vscode-logging/logger": "1.2.2",
     "@xml-tools/ast": "5.0.0",
     "lodash": "4.17.21",
-    "node-fetch": "3.2.10",
+    "node-fetch": "2.6.9",
```

Comment:
Why did we downgrade node-fetch version?

Reply:
Unfortunately jest doesn't support ES6 imports (node-fetch V3 produced in ES6). Node-fetch V2 was compiled in CommonJS, and works fine with jest. The issue is related to this code https://github.com/SAP/ui5-language-assistant/blob/0343f45aa03f47808c9408737d496a9dc66f6af4/packages/logic-utils/src/utils/fetch.ts#L18